SU-SOE/stanford_soe_helper
# 7.x 2.x news spotlight views

# READY FOR REVIEW

# Summary
- Adds support for related departments for horizontal views
- Adds spotlight/faculty and spotlight/students 
- Removes 2 item, span4 vertical view.

# Needed By (Date)
- Yesterday

# Criticality
- Yup

# Steps to Test
- create several spotlight nodes with both faculty and student affiliation, and one or more departments
- Verify that that /spotlight/faculty and /spotlight/students display faculty and student nodes appropriately
- Verify that /spotlight displays all nodes
- Place horizontal view block and verify that it displays department related nodes appropriately

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2014
https://stanfordits.atlassian.net/browse/SOE-2015
https://stanfordits.atlassian.net/browse/SOE-2063
https://stanfordits.atlassian.net/browse/SOE-2064

## Related PRs
Encapsulates:
https://github.com/SU-SOE/stanford_soe_helper/pull/86

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
